test/extensions/filters/network/wasm/config_test.cc

```diff
@@ -176,6 +176,93 @@ TEST_P(WasmNetworkFilterConfigTest, FilterConfigFailOpen) {
   EXPECT_EQ(filter_config.createFilter(), nullptr);
 }
 
+TEST_P(WasmNetworkFilterConfigTest, FilterConfigABIUnrestrictedByDefault) {
+  if (GetParam() == "null") {
+    return;
+  }
+  const std::string yaml = TestEnvironment::substitute(absl::StrCat(R"EOF(
+  config:
+    vm_config:
+      runtime: "envoy.wasm.runtime.)EOF",
+                                                                    GetParam(), R"EOF("
+      code:
+        local:
+          filename: "{{ test_rundir }}/test/extensions/filters/network/wasm/test_data/test_cpp.wasm"
+    capability_restriction_config:
+      allowed_abi_functions:
```

Comment:
I think this needs to be `allowed_abi_functions: {}` for empty value.

Reply:
Fixed.